yellowbrick/gridsearch/pcolor.py

```diff
@@ -0,0 +1,225 @@
+"""
+Colorplot visualizer for gridsearch results.
+"""
+
+import time
+import numpy as np
+
+from .base import GridSearchVisualizer
+from ..exceptions import YellowbrickValueError
+
+from sklearn.metrics.pairwise import pairwise_distances
+from sklearn.preprocessing import LabelEncoder
+from matplotlib.mlab import griddata
+
+
+## Packages for export
+__all__ = [
+    "GridSearchColorPlot",
+    "gridsearch_color_plot"
+]
+
+
+##########################################################################
+## Dimension reduction utility
+##########################################################################
+
+def param_projection(cv_results, param_1, param_2):
+    """
+    Projects the grid search results onto 2 dimensions.
+
+    The display value is taken as the max over the non-displayed dimensions.
+
+    Parameters
+    ----------
+    cv_results : dict
+        A dictionary of results from the `GridSearchCV` object's `cv_results_`
+        attribute
+
+    param_1 : string
+        The name of the parameter to be visualized on the horizontal axis.
+
+    param_2 : string
+        The name of the parameter to be visualized on the vertical axis.
+
+    Returns
+    -------
+    ax : matplotlib axes
+        Returns the axes that the classification report was drawn on.
+    """
+    # Get unique values of the two display parameters
+    x_vals = sorted(list(set(cv_results['param_' + param_1].compressed())))
+    y_vals = sorted(list(set(cv_results['param_' + param_2].compressed())))
```

Comment:
Ok, so just to make sure I understand what's happening here - you're fetching the param mask from the cv results dictionary, compressing it to get the real values instead, then calling set to find all unique values. These are the categories for the grid, right? 

I've got a couple of notes:

1. `'param_' + param_1` is used twice, it might be easier to store these in variables and note what they are or just fetch the mask array from the results and store that. 
2. If the parameter doesn't exist in the grid search, this will return a `KeyError` - it would be better if we raised a yellowbrick key error, e.g. `YellowbrickKeyError(YellowbrickError, KeyError)` that says "parameter not found in grid search" so it's more obvious to the user what happened.

Reply:
Yes, that's right - I've added a couple comment lines to try to clarify and stored the masked array like you suggested in (1).  Also did what you suggested in (2).